README.md

```diff
@@ -345,6 +345,24 @@ included by default unless you choose which one to exclude with the
 then you have to specify each backend that you want individually. You can get
 the full list of these flags with the `--help` flag.
 
+#### --auto-config-uri
+This is a special URI which if set, will try and pull a config from that
+location on startup. It will use the cookie file stored at
```

Comment:
> Looking around, I think I see that a difference is that the `config.json` at `--config-path` can have a `auto-config-uri` but possibly the `config.json` at that URI can't?

I was wrong here. The `config.json` at the URI *can* have an `auto-config-uri` with this implementation, and that would presumably point to itself so that each time the tool is run it would check for updates and overwrite your config if it's changed.

That also explains the `auto-` because it will stay configured that way as long as you use the same `config-path`.

Reply:
You should ignore most of this conversation. It was written early, based on mistaken guesses at how this works, before I read the rest of the code to learn how it *really* works. Actually I thought this conversation had gotten lost, or I would have deleted it before submitting.

Reply:
`config-path` is about specifying where you config actually goes. This is useful if you want to have an alias such as:

```
alias yesiscan_somecategory='yesiscan --config-path=...'
```

The auto-config stuff is about pulling down a standardized config file to whatever path you want.

Reply:
> You should ignore most of this conversation. It was written early, based on mistaken guesses at how this works, before I read the rest of the code to learn how it really works. Actually I thought this conversation had gotten lost, or I would have deleted it before submitting.

Already replied, sorry! If you would like a walkthrough of some of the code like a mini-hacking session, lmk!

Reply:
No need. I read the code and understood it. I'm not sure why I couldn't see that these comments were still here. I was switching between inline and side-by-side so maybe it became invisible in one or the other.